hackathon_site/hardware/api_urls.py

```diff
@@ -1,16 +1,20 @@
-from django.urls import path, re_path
+from django.urls import path, re_path, include
 from hardware import views
+from rest_framework import routers
 
 
 app_name = "hardware"
 
+router = routers.DefaultRouter()
+router.register(r"orders", views.OrderViewSet, basename="order")
```

Comment:
Is this absolutely necessary because of the view set? If so, might as well convert the rest of the url patterns here into the router so we stay consistent.

Reply:
reverted back to APIView